.travis.yml

```diff
@@ -7,6 +7,23 @@ matrix:
   include:
   - jdk: oraclejdk7
     script: mvn -q deploy --settings .travis-settings.xml -Dno.gem.deploy=true
+  - jdk: oraclejdk7
+    env: ANDROID_EXAMPLES=true
+    language: android
+    android:
```

Comment:
No, they are ordered as the are for no good reason at all (from copying the definition of the old Android job mostly).